### PR TITLE
bump linuxkit version, init version, place etc/issue in good place, fix linuxkit_cmdline parsing in make-roots

### PR DESCRIPTION
**Pull request by deitch**
Several small changes in this PR, split across two commits, as each stands on its own. These also help prepare the groundwork for the installer rearchitecture, which almost is done, and should have a real PR in the coming days.

* moves `/etc/issue` - which contains EVE logo and name on login - to `pkg/eve/installer/etc_issue`, where it gets placed in the `dist/<arch>/<version>/installer/` directory, and then placed in the rootfs by a `files` entry in `rootfs.yml.in`. This makes it easier to find and edit, and more importantly, reusable in other builds.
* bumps linuxkit version to v1.5.1, which fixes some issues with read-only volumes
* bumps linuxkit init to the latest. This changes the default cgroups version to v2, which breaks a number of things in pillar and even startup, so also adds the cmdline flag to force cgroups v1. However, the `make-rootfs` scripts that consumed that had a bug, so fixed those as well.

I could have held off with these until installer PR is ready, but these are much better on their own. It makes the installer PR smaller and simpler, and if there are issues with the installer PR, these will not be affected.

### CODE REVIEW

File: images/rootfs.yml.in
Comment (eriknordmark) [inline images/rootfs.yml.in:4]: @rouming will this impact your RT work? I think you talked about cgroupv2.

Comment (deitch) [reply images/rootfs.yml.in:4]: Shouldn't be a problem. Everything now is v1, this makes it explicit. Actually not only does pillar have an issue with v2, runc needs some kernel settings that we don't have enabled. @christoph-zededa knows about them I think. 

Either way, when we are ready, just remove that line and it runs v2. You even can do it interactively at grub time.

Comment (rouming) [reply images/rootfs.yml.in:4]: We do not support v2, so as @deitch said, this makes it explicit. I'm fine with the change.

**Review by rouming — APPROVED** [review]

**Comment by deitch** [comment]
Just tried rerunning a failing test, to see if it was transient. Unlikely, since all of the eden tests failed, but worth trying. No luck, it still failed at the same point.

**Comment by deitch** [comment]
I had made an error in the PR. That is what CI is for! Needs a new approval to kick off eden tests again, please.

**Review by rouming — APPROVED** [review]

**Comment by OhmSpectator** [comment]
I really like the PR description; it explains the changes in the way I would like to see it in the commit messages. But the commit messages are empty =( Once the PR is merged this valuable info will be lost and not appear in the git history =(

**Comment by deitch** [comment]
@OhmSpectator those are fair comments. I don't want to make a change to the commits - even just messages - in a way that will cause the whole long CI to run again. If these pass (still have 2 spurious failures, which I restarted), then once those are green, I will change the commit messages, push out, and merge immediately; no need to wait for full hours-long CI if the only change is the commit message content.

If I have to make changes because of actual failures, I will do it then.

**Comment by deitch** [comment]
Finally, all clean. I will update the commit messages and then merge.

**Comment by deitch** [comment]
Done @OhmSpectator , much more detailed commit comments. Nothing changed in code, so I am just going to merge this in. CI was 100% green before.

**Pull request merged**
